In [6]:
import sys

In [5]:
sys.version

'3.11.14 | packaged by Anaconda, Inc. | (main, Oct 21 2025, 18:30:03) [MSC v.1929 64 bit (AMD64)]'

In [7]:
sys.executable

'c:\\Users\\dil91\\anaconda3\\envs\\ds_env\\python.exe'

In [8]:
import pandas as pd

In [3]:
import numpy as np

In [9]:
df = pd.DataFrame({
    "iteration": [1, 2, 3],
    "status": ["PASS", "PASS", "FAIL"]
})
df

iteration status
0          1   PASS
1          2   PASS
2          3   FAIL

☐ Conda env created (ds_env)

☐ Python 3.11 active

☐ ipykernel installed

☐ Notebook runs without prompts

☐ NumPy & Pandas import OK

everything done via VS code, Conda prompt

Created  env in VS code terminal:

        conda create -n ds_env python=3.11 -y

Changed terminal to powershell in VS code to use the conda environment

        conda activate ds_env

install ipykernel

        conda install ipykernel -y

install packages

        conda install numpy pandas jupyter -y